src/main/java/org/pac4j/vertx/handler/impl/Pac4jAuthHandlerOptions.java

```diff
@@ -26,6 +26,7 @@
     private String clientName = "";
```

Comment:
Your feedbacks are always valuable: nothing better than peer programming on advanced issues! That said, the names are not a big deal. The customization of the ProfileManager instead of the WebContext is a real improvement.